src/app/content/components/BookBanner.tsx

```diff
@@ -66,14 +79,22 @@ const BookTitle = styled.a`
   :hover {
     text-decoration: underline;
   }
+
+  ${ifMiniNav(css`
+    width: 27rem;
+
+    ${theme.breakpoints.mobile(css`
+      display: none;
+    `)}
+  `)}
 `;
 
 // tslint:disable-next-line:variable-name
-const BookChapter = styled.h1`
-  ${h3Style}
+const BookChapter = styled((props) => props.variant === 'mini' ? <h2 {...props} /> : <h1 {...props} />)`
```

Comment:
Why does the heading level change based on whether the `mini` is visible or not? It seems odd that the root of the heading tree (`H1`) would change based on the scroll position.

Reply:
it doesn't change, the mini and big both exist in the dom at the same time, so the mini is not an h1 because there can only be 1 h1

Reply:
Hmm, in that case it seems like only one of these is visible at any time. If that is the case, then should the visible one be `<h1>` while the invisible is switched to be `<h2>`?